echopype/convert/set_groups_azfp.py

```diff
@@ -208,12 +212,12 @@ def set_platform(self) -> xr.Dataset:
                 ),
                 "vertical_offset": (
                     ["time2"],
-                    [np.nan] * len(time2),
+                    depth,
                     self._varattrs["platform_var_default"]["vertical_offset"],
                 ),
                 "water_level": (
-                    [],
-                    np.nan,
+                    ["time1"],
```

Comment:
water_level is a scalar variable. It has no dimension.
```suggestion
                    [],
```

Reply:
Updated!